packages/client/src/components/DropzoneContainer/DropzoneContainer.tsx

```diff
@@ -40,31 +49,35 @@ export const DropzoneContainer: FC<DropzoneContainerProps> = memo(
 				) : (
 					<>
 						<Icon iconName="Upload" />
-						Upload dataset
+						{contentText}
 					</>
 				)}
 				<Container>
 					<ChildContainer>{children}</ChildContainer>
 				</Container>
-			</DragFilesContainer>
+			</DragFilesArea>
 		)
 	},
 )
 
 const LoadingText = styled.span`
 	margin-left: 4px;
 `
-const DragFilesContainer = styled(DefaultButton)<{
+const DragFilesArea = styled(DefaultButton)<{
 	isDragging: boolean
+	isButton: boolean
 }>`
 	display: flex;
 	white-space: nowrap;
 	margin: 8px;
-	border: 1px solid
-		${({ theme, isDragging }) =>
-			isDragging ? theme.application().accent : theme.application().foreground};
 	color: ${({ theme, isDragging }) =>
 		isDragging ? theme.application().accent : theme.application().foreground};
+	opacity: ${({ isDragging }) => (isDragging ? 0.5 : 1)};
+	border: 1px
+		${({ theme, isButton }) =>
+			!isButton
+				? 'dashed ' + theme.application().accent()
```

Comment:
You need to call the `hex` function on each of these Color objects for it to work with css